### DISCUSSION TITLE
🚀 Add file location (subfolder) to "File Edit" information (EE7)

### DISCUSSION THREAD

**Discussion by jcogs-design** [ExpressionEngine/ExpressionEngine#3812] [Feature Requests]
### Suggest an idea for ExpressionEngine

When viewing a file selected from the Files Area it would be helpful if it reported its location within the File directory / folder structures. Currently you can find a file by looking at "all files" list, but not easily work out where it is living within the Files area. vis.
![image](https://github.com/ExpressionEngine/ExpressionEngine/assets/13821249/cf4d6127-c66d-41bb-9717-6dd5b914b3f5)

**Comment by intoeetive** [comment]
@jcogs-design this information is already present in the breadcrumbs though

![image](https://github.com/ExpressionEngine/ExpressionEngine/assets/752126/c2748677-b028-4ab7-9184-8e0bcd7a0dbb)

**Reply by jcogs-design** [reply]
Indeed - but as noted previously the breadcrumb trail does not include any information about sub-folders. But actually that's important to know too.

**Comment by jcogs-design** [comment]
The breadcrumb trail does not include information about sub-folders :(

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/3906
